src/state/store/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { createStore } from "redux";
+import rootReducer from "../reducers/rootReducer";
+
+const configureStore = () => {
+  return createStore(rootReducer);
+};
+
+export default configureStore;
```

Comment:
I would go for creating and exporting the `store` directly here, and then use it in your app AND your services (as i demonstrated the other day)

Reply:
Fixed

Reply:
No, it is not ´fixed´

Reply:
it is fixed. maybe it's just something wrong with github